.github/actions/container_sbom/action.yml

```diff
@@ -21,7 +21,7 @@ runs:
       id: checkout
       uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
       with:
-        ref: ${{ inputs.ref || github.head_ref }}
```

Comment:
This is not a correct solution. The release pipeline (that calls this indirectly) depends on the ref used during checkout being different from github.head_ref

Reply:
You would have to pass inputs.ref all the way down here or perform the checkout in each workflow before calling the action

Reply:
I think the checkout is actually not needed here, since the parents should already do the checkout before (the action is buried few layers deep). They probably should do this in any case, I missed the levels of depth the action is called here.

Removed it.